test/e2e/leaderelection.go

```diff
@@ -34,6 +34,10 @@ var _ = Describe("Leader Elector", func() {
 	tl := framework.NewE2ELogger()
 
 	It("should chose secondary instance, primary goes down", func() {
+		if framework.TestContext.LimitedScope {
+			framework.Skipf("Leader Elector is not need to be tested in namespaced federation setup")
```

Comment:
nit: s/Leader Elector.*/Testing of leader election requires an isolated test namespace which is only possible with cluster-scoped federation/